packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -82,6 +82,42 @@
                                     >-</span
                                 >
                             </div>
+                            <div data-ng-if="flavor.technicalBlob.gpu">
```

Comment:
Could this be in the `Flavor` model ? Something like `hasGpu()` ?

Reply:
This is done. Thanks :)